polkadot/xcm/xcm-simulator/src/mock_message_queue.rs

```diff
@@ -41,9 +48,11 @@ pub mod pallet {
 	pub struct Pallet<T>(_);
 
 	#[pallet::storage]
+	#[pallet::getter(fn parachain_id)]
 	pub(super) type ParachainId<T: Config> = StorageValue<_, ParaId, ValueQuery>;
 
 	#[pallet::storage]
+	#[pallet::getter(fn received_dmp)]
```

Comment:
I know that storage getters were on their way out https://github.com/paritytech/polkadot-sdk/issues/3326, let's not add more here